ros2node/test/test_node.py

```diff
@@ -34,3 +36,28 @@ def test_parse_node_name():
     assert name.full_name == '/ns/talker'
     assert name.namespace == '/ns'
     assert name.name == 'talker'
+
+
+def test_duplicate_names():
```

Comment:
Any sanity checking needed, e.g., empty list input to ```annotate_duplicate_nodes```? What about using more than 2 nodes?